index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, shrink-to-fit=no"
+    />
+
+    <!-- Page description -->
+    <meta
+      name="description"
+      content="Switch to smart house company offers at an affordable price the opportunity to buy software and installation services. They allow, at the stage of building a house, to create a friendly and functional place that can be controlled from anywhere in the world."
+    />
+
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+
+    <!-- Fontawesome icons -->
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="./styles/style.css" />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Switch</title>
+  </head>
+
+  <body class="d-flex align-items-center justify-content-end position-relative">
+    <div class="container d-flex flex-column position-relative">
+      <!-- Logo -->
+      <a class="d-flex" href="index.html"
+        ><img
+          class="logo ml-auto"
+          src="./assets/images/logo.png"
+          alt="Company logo"
+      /></a>
```

Comment:
```suggestion
      <a class="d-flex" href="index.html">
           <img
          class="logo ml-auto"
          src="./assets/images/logo.png"
          alt="Company logo"
      />
 </a>
```

Zachowałabym zamykanie tagu w tej samej  linii  😉